packages/components/src/font-size-picker/next/use-font-size-control.js

```diff
@@ -98,37 +83,20 @@ export function useFontSizeControl( props ) {
 
 	const isDefaultValue = ! is.defined( value );
 
-	const classes = cx( styles.FontSizeControl, className );
```

Comment:
`className` is actually a requirement, it comes from `useContextSystem` and automatically applies the `wp-component-` and `component-` classes:

https://github.com/ItsJonQ/g2/blob/master/packages/context/src/use-context-system.js#L54

Reply:
Thank you @saramarcondes, I was not aware that className came from useContextSystem. I reverted the change 👍